arctic/store/versioned_item.py

```diff
@@ -1,19 +1,20 @@
 from collections import namedtuple
 
 
-class VersionedItem(namedtuple('VersionedItem', ['symbol', 'library', 'data', 'version', 'metadata'])):
+class VersionedItem(namedtuple('VersionedItem', ['symbol', 'library', 'data', 'version', 'metadata', 'info'])):
```

Comment:
A lot of the change comes about due to the addition of `info` to the API here.  I was wondering if it might be better just to have a method `get_info` on the `VersionStore` and let clients that care use that?